charts/kube-prometheus-stack/values.yaml

```diff
@@ -60,6 +60,8 @@ defaultRules:
 
   ## Runbook url prefix for default rules
   runbookUrl: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#
+  ## Thresholds for KubeQuotaAlmostFull alert
+  kubernetesResourceAlmostFullQuotaCondition: ' > 0.9 < 1'
```

Comment:
Since we're starting a new paradigm here, do you think it would be a good idea to have the values in a separate block structure and also match them to the alert name itself?

Reply:
@vsliouniaev thanks for suggestion. I extended the rules sync script to support the notion of `alert_expressions_replacement_map ` that be used to override alert expressions only:
https://github.com/prometheus-community/helm-charts/pull/1372/files#diff-90d0cb7c99a0aca67494fd6a5957bac966b4a6a7a7e5fb59759251d7e9ca2fbbR151

LMK